src/pages/alerted/alerted.tsx

```diff
@@ -38,7 +38,9 @@ const Alerted = () => {
         return `${t('name', {lng: 'en'})}: ${currentValue.name}
                 ${t('phone_number', {lng: 'en'})}: ${currentValue.phoneNumber}
                 ${t('emergency', {lng: 'en'})}: ${emergency}
-                ${t('location', {lng: 'en'})}: ${currentValue.address}
+                ${t('address', {lng: 'en'})}: ${currentValue.address}
+                ${t('location', {lng: 'en'})}: ${currentValue.geolocation!.latitude}, ${currentValue.geolocation!.longitude}
+                ${t('location_accuracy', {lng: 'en'})}: ${Math.floor(currentValue.geolocation!.accuracy)} m
                 ${t('comment', {lng: 'en'})}: ${currentValue.addressComment}
                 `
     }, [currentValue.address, currentValue.addressComment, currentValue.emergencyCode, currentValue.name, currentValue.phoneNumber, t]);
```

Comment:
you need to fix dependencies otherwise the build is broken

Reply:
Done.